db/seeds.rb

```diff
@@ -59,6 +63,20 @@
 bridge_builders2 = Tenant.find(2)
 bridge_builders3 = Tenant.find(3)
 
+Tenant.find(1).users.create!(
+  username:               "borrower",
+  first_name:             "first",
+  last_name:              "last",
+  email:                  "borrower@borrower.com",
+  password:               "password",
```

Comment:
In my seed file, I created a user and then did user.tenant.create!(....) ... it's the same thing right? I created a tenant off a user whereas you created a user off of a tenant. There's no difference right? At the end of the day we have a borrower correct?